luda-adventure/src/app/game/game.rs

```diff
@@ -0,0 +1,133 @@
+use super::{player_character::new_player, *};
+use namui::prelude::*;
+
+pub struct Game {
+    pub player_entity_id: Uuid,
```

Comment:
You mean, don't use `const SOME_ID = "12d12-..."`.

Reply:
No, just like this

```rust
pub struct Game {
  ...
  pub quest_id: Uuid,
}

implk Game {
  pub fn new() -> Self {
    ...
    let quest_id = namui::uuid();
    
    Self {
      quest_id,
    }
  }
}
```

And then, it would be better to save quest id in json file, not code. Are you planing making all resources in code?

Reply:
Why do you think it's better?

Actually I will save resources to your code. I don't think there's any reason to save it in the json file because I didn't plan to make an editor.

Reply:
Then if somebody wants to make a quest then they need to read and write code?

Reply:
I mean, above what i'm saying is saving data in external file,

And What I want to say is that's hard-coding which generally one of the code-smell. There are very much of reason why it's not a good idea to make a hard-coding.

What I suggest is that making it flexible anytime that Id can be loaded from file or just generated from code like my above example code.

Reply:
Who will make a quest?

Reply:
I think I will

Reply:
I see. Anyway it's not a that much big deal but I think const id isn't good option for flexibility of code and putting it to state is better. Go a head what do you want to do.